### DISCUSSION TITLE
MsQuic + Hill Cipher + Network Coding

### DISCUSSION THREAD

**Discussion by scopedog** [microsoft/msquic#3925] [Show and tell]
I have implemented Augmented Hill Cipher and Random Network Coding to MsQuic and wrote a research paper on it.
The paper got the Best Paper Award in two categories at IEEE UEMCON 2023 and will appear on IEEE Xplore soon.
It is downloadable from https://www.researchgate.net/publication/374776143_Secure_Fast_and_Loss-Tolerant_Communication_with_Hill_Cipher_and_Network_Coding.
Please take a look at it.
I will open-source some of my programs soon (if my company doesn't change their mind).

**Comment by nibanks** [comment]
Sounds great! Congrats! We'll definitely take a look.
